app/addons/dashboard/stores.js

```diff
@@ -0,0 +1,169 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'addons/activetasks/actiontypes'
+], function (app, FauxtonAPI, ActionTypes) {
+
+  var DashboardStore = FauxtonAPI.Store.extend({
+
+    init: function (collectionTable, backboneCollection) {
```

Comment:
This looks good. But we were all just chatting + agree that having a `reset()` method on all stores is a handy thing - partly just for tests so we can always clear them out after a mocha test is run. Mind moving the content of this method into a `reset()` method and call it from `init()`?

Oh, and one other thing. So `initialize` has special meaning for a store within Fauxton: it's automatically called when the store is first created. I'll defer to @michellephung and @robertkowalski on this, but I kind of prefer avoiding the name `init` for a store method because I tend to confuse `init` with `initialize` (I've looked it up like 3 times already). Maybe name it `setup` instead?